src/client/src/utils/session.js

```diff
@@ -1,31 +1,31 @@
-export const register = user =>
-  fetch('http://some_backend:8080/login/register', {
-    method: 'POST',
-    body: JSON.stringify(user),
-    headers: {
-      'Content-Type': 'application/json'
-    }
-  })
-
-export const login = user =>
-  fetch('http://some_backend:8080/login', {
-    method: 'POST',
-    body: JSON.stringify(user),
-    headers: {
-      'Content-Type': 'application/json'
-    }
-  })
-
-export const logout = () => fetch('http://some_backend:8080/logout', { method: 'DELETE' })
-
-export const checkLoggedIn = async preloadedState => {
-  const response = await fetch('http://some_backend:8080/login')
-  const { user } = await response.json()
-  preloadedState = {}
-  if (user) {
-    preloadedState = {
-      session: user
-    }
-  }
-  return preloadedState
-}
+export const register = user =>
+  fetch('http://localhost:18080/login/register', {
```

Comment:
Czy zmiana adresu z komunikacji pomiędzy kontenerami na localhosta nie spowoduje, że każdy musi mieć apkę dostępną pod locahost, a nie tak jak w przypadku docker toolbooxa pewne ip (bez przekierowań)?

Reply:
Na pewno spowoduje, że projekt zacznie działać. Jak masz przekierowane porty tak jak trzeba to wszystko będzie grać. Inaczej nie działa bo testowaliśmy to z Ciupsonem.

Reply:
to depends_on w docker-compose nie działało?

Reply:
Na gałęzi frontend zapakowałem adres w zmienną "url" aby jakiekolwiek zmiany oddbywały się w jednym miejscu. Jeśli będziemy chcieli to możemy skorzystać z "https://www.npmjs.com/package/http-proxy-middleware" (chociażby na devie), nie będziemy musieli wtedy podawać urla backendu w całym reakcie.